style-guide/styles/wds-components/_toggles.scss

```diff
@@ -0,0 +1,76 @@
+@import '../wds-variables/colors';
+
+.wds-switch {
+	&__input {
+		display: none;
+	}
+
+	&__label {
+		cursor: pointer;
+		font-size: 16px;
+		padding: 7px 5px 7px 52px;
+		position: relative;
+
+		&:empty {
+			padding-left: 45px;
+		}
+
+		&:before, &:after {
+			content: '';
+			margin: 0;
+			outline: 0;
+			position: absolute;
+			top: 50%;
+			transform: translate(0, -50%);
+			transition: all 0.3s ease;
+		}
+
+		&:before {
```

Comment:
Please use `::` for pseudo-elements: https://www.w3.org/TR/css3-selectors/#pseudo-elements